src/main/java/com/flightstats/hub/app/StorageBackend.java

```diff
@@ -0,0 +1,15 @@
+package com.flightstats.hub.app;
+
+public enum StorageBackend {
+    AWS("aws"),
+    NAS("nas"),
+    TEST("test");
```

Comment:
This is for my understanding. What does these values mean? AWS means S3? NAS means spoke storage?

Reply:
FWIW, when the value differs from the name only by case, I just use the appropriately cased name. E.g:
```
public enum StorageBackend { aws, nas, test; }
```

Goes a little against the uppercase convention for constants, but I don't think of enum at the same conceptual level as constants. You can even think of the enum values as "public final" instance variables (which aren't upper cases) of the StorageBackend class if that helps.

Mostly, I ask myself "Is this extra string that differs only by case adding value?" If the answer is no, then I ignore the uppercase convention. If the answer is yes, then I tend to uppercase them since the real valuable information is the parameters they're instantiated with.

Reply:
@manjula-chikkanagappa aws == s3, nas == network attached storage (I always forget what that acronym is)

Reply:
glad to hear about the casing convention because the upper-casing was giving me problems in tests